vayesta/core/scmf/scmf.py

```diff
@@ -100,7 +100,12 @@ def kernel(self, *args, **kwargs):
 
             dm1 = self.mf.make_rdm1()
             # Check symmetry
-            self.emb.check_fragment_symmetry(dm1)
+            try:
+                self.emb.check_fragment_symmetry(dm1)
+            except:
```

Comment:
Could you please define a `SymmetryError` in `core/util.py`, change the exception raised in `check_fragment_symmetry` to `SymmetryError` and catch only `SymmetryError` exceptions here - thanks